tests/ckeditor/basepath.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals beforeEach, describe, it, expect, CKEDITOR, window, document */
+
+'use strict';
+
+beforeEach( function() {
+	// Ensure that no CKEDITOR_BASEPATH global is available.
+	delete window.CKEDITOR_BASEPATH;
+
+	// Remove all script elements from the document.
+	removeScripts();
+} );
+
+describe( 'ckeditor.basePath', function() {
+	it( 'Full URL', function( done ) {
+		CKEDITOR.require( [ 'ckeditor' ], function( CKEDITOR ) {
+			addScript( 'http://bar.com/ckeditor/ckeditor.js' );
+			expect( CKEDITOR._getBasePath() ).equals( 'http://bar.com/ckeditor/' );
```

Comment:
> .to.equal() is just two letters longer but reads way better, just sayin'...

I know, but while writing my comment I noticed that it's not going through my fingers too well :P

Anyway - `to.equal()` is not bad. I'll perhaps use to type it so it will be faster. Just saying so we remember to keep them short.